ducktape/mark/_mark.py

```diff
@@ -190,10 +190,33 @@ def __eq__(self, other):
         return super(Parametrize, self).__eq__(other) and self.injected_args == other.injected_args
 
 
+class Env(Mark):
+    def __init__(self, **kwargs):
+        self.injected_args = kwargs
+        self.should_ignore = False
+        for key, value in kwargs.iteritems():
```

Comment:
something like `self.should_ignore = any(os.environ[key] == value for key,value in kwargs.iteritems())` would probably be more idiomatic if you want to tighten this up a bit.